build/tools/CFamilyJarPreProcessor/CFamilyJarPreProcessor.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Separate solution to simplify debugging. The exe project is referenced in the main solution.